static/scss/partials/main.scss

```
@@ -849,40 +849,27 @@ input:focus::placeholder {
     opacity: .5;
 }
 
+.is-visible-with-addon {
+    // This class hides an element anything in the regular website;
+    // however, the add-on injects a stylesheet that reveals elements with this class.
+    display: none;
+    visibility: collapse;
+}
 
-
-.dashboard-banners.invisible {
-    transition: all .2s ease;
-    opacity: 0;
-    padding: 0;
-    max-height: 0;
+.is-hidden-with-addon {
+    // This class doesn't do anything in the regular website;
+    // however, the add-on injects a stylesheet that hides elements with this class.
 }
 
+
 .dashboard-banners.drop-shadow {
     box-shadow: 0 12px 18px 2px rgba(34,0,51,.04),0 6px 22px 4px rgba(7,48,114,.12),0 6px 10px -4px rgba(14,13,26,.12);
 }
 
-.banner-content.invisible {
-    max-height: 0;
-}
-
 .dashboard-banners {
-    width: 100%;
-    opacity: 1;
-    padding: 1em 0;
-    transition: all .3s ease;
+    // width: 100%;
```

nit: remove commented code 
```suggestion
    // width: 100%;
```

---

Hmm that's weird, I wonder how that got there :sweat_smile: Fixed.